WMF Framework/EventLoggingService.swift

```diff
@@ -494,7 +459,6 @@ public class EventLoggingService : NSObject, URLSessionDelegate {
     }
     
     private func resetInstall() {
-        appInstallID = nil
```

Comment:
Aha, will do.  I guess I was working with mistaken assumptions about what the "app install ID" is supposed to represent.  It's surprising to me that it changes with the opt-in switch being toggled off and on again; I would think it would in fact survive for the lifetime of the app installation regardless of analytics opt-in state.  But FWIW, the Android app does the same, and this wouldn't really be the time or place to change that even if we wanted to, I suppose.

Reply:
@mdholloway yeah true, I would think install ID would be generated and persist regardless of the enable toggle state. Good to know that Android works the same way though.